packages/airnode-deployer/src/cli/commands.ts

```diff
@@ -91,6 +95,15 @@ export async function deploy(configPath: string, secretsPath: string, receiptFil
   fs.rmSync(tmpDir, { recursive: true });
 
   writeReceiptFile(receiptFile, mnemonic, config, output, deploymentTimestamp);
+
+  if (deploymentError) {
+    logger.fail(
+      `Airnode deployment failed due to errors.\n` +
```

Comment:
```suggestion
      `Airnode deployment failed due to an unexpected error.\n` +
```

Reply:
There might me multiple errors actually (e.g. yesterday I got 3 for GCP deployment). Would you still consider your suggestion more accurate?

Reply:
Ok I didn't realise that. Maybe just "due to unexpected errors". 

Just "due to errors" sounds a bit weird to me